vendor/knative.dev/eventing/pkg/kncloudevents/event_dispatcher.go

```diff
@@ -125,10 +126,12 @@ type senderConfig struct {
 
 type Dispatcher struct {
 	oidcTokenProvider *auth.OIDCTokenProvider
+	clientConfig      eventingtls.ClientConfig
 }
 
-func NewDispatcher(oidcTokenProvider *auth.OIDCTokenProvider) *Dispatcher {
+func NewDispatcher(clientConfig eventingtls.ClientConfig, oidcTokenProvider *auth.OIDCTokenProvider) *Dispatcher {
```

Comment:
This is a breaking change, introduced by https://github.com/knative/eventing/pull/7532/files

and causing trouble on the `pkg/source/adapter/adapter.go` file